### PR TITLE
Merge Feature/dashboard to develop

### PR DESCRIPTION
**Pull request by AkuaQ**
The dashboard map page has been created.

### CODE REVIEW

File: ParkIt/ParkIt/DashboardViewController.swift [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+    /*
+    // MARK: - Navigation
+
+    // In a storyboard-based application, you will often want to do a little preparation before navigation
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        // Get the new view controller using segue.destination.
+        // Pass the selected object to the new view controller.
+    }
+    */
```
Comment (AMadanlal) [inline ParkIt/ParkIt/DashboardViewController.swift]: can remove this if not needed

**Review by AMadanlal — APPROVED** [review]
Looks good

**Review by Tiewhan — APPROVED** [review]
Please give an image of the UI changes made. It makes reviewing changes much easier

**Comment by AkuaQ** [comment]
Dashboard:
Image 2020-03-12 at 07 47

**Pull request merged**